src/demo-app/baseline/baseline-demo.html

```diff
@@ -14,9 +14,12 @@
       <input mdInput placeholder="Input" value="Text Input">
     </md-form-field>
     | Text 5 |
-    <md-select placeholder="Select">
-      <md-option value="option">Option</md-option>
-    </md-select>
+    <md-form-field>
```

Comment:
In the past we've preferred composition style APIs since they're more flexible, even if they are a bit more verbose. I think the extra typing is worth it, but it's a fair point. @jelbourn do you have an opinion?